examples/synthetic_data/syn_text_pii_scanner_example.ipynb

```diff
@@ -0,0 +1,608 @@
+{
```

Comment:
<p>Be more explicit in what the reader should learn. That is, the NERLongFormer, fine tuned on the TAB data, is good at identifying the entities.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/LeakPro/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>